### ISSUE TITLE
Updater.__init__() got an unexpected keyword argument 'token'

### ISSUE THREAD

**Issue by maksimaee** [markusressel/cli2telegram#18]
Getting this error message when running this command printf "Message" | cli2telegram -b "123456789:ABCDEFG" -c "123456789" with valid bot token and chat id. System has python 3.10.9 and export PATH=$PATH:$HOME/.local/bin in .bash_profile:

Updater.__init__() got an unexpected keyword argument 'token'
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.10/site-packages/cli2telegram/util.py", line 94, in _try_send_message
    updater = Updater(token=bot_token, use_context=True)
TypeError: Updater.__init__() got an unexpected keyword argument 'token'
Error sending message, retrying in 10.0 seconds...

I've tried editing util.py with valid token and running cli2telegram with  updater = Updater("123456789:ABCDEFG") in util.py and I get the following error:
Updater.__init__() missing 1 required positional argument: 'update_queue'
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.10/site-packages/cli2telegram/util.py", line 94, in _try_send_message
    updater = Updater("123456789:ABCDEFG")
TypeError: Updater.__init__() missing 1 required positional argument: 'update_queue'
Error sending message, retrying in 10.0 seconds...

**Comment by markusressel** [comment]
Hey there, thx for the report! :rocket: 

Make sure to **not** install python-telegram-bot v20, which is a massive breaking change. Instead downgrade to the last v13 version (currently [v13.15](https://github.com/python-telegram-bot/python-telegram-bot/releases/tag/v13.15)).

This project currently doesn't pin dependencies in a good way, so you probably got the latest v20 version. To improve this we could use a poetry configuration, instead of the `requirements.txt`.

**Comment by maksimaee** [comment]
Thank you for getting back to me so quickly. I have downgraded python-telegram-bot to v13.15 and now getting this error:

emojize() got an unexpected keyword argument 'use_aliases'
Traceback (most recent call last):
  File "/home/maksima/.local/lib/python3.10/site-packages/cli2telegram/util.py", line 95, in _try_send_message
    send_message(bot=updater.bot, chat_id=chat_id, message=message, parse_mode="markdown")
  File "/home/maksima/.local/lib/python3.10/site-packages/cli2telegram/util.py", line 43, in send_message
    emojized_text = emojize(message, use_aliases=True)
TypeError: emojize() got an unexpected keyword argument 'use_aliases'
Error sending message, retrying in 10.0 seconds...

Here is pip output:

`[~]$ pip check
No broken requirements found.
[~]$ pip show cli2telegram

Name: cli2telegram
Version: 2.0.0
Summary: Small utility to send Telegram messages from the CLI.
Home-page: https://github.com/markusressel/cli2telegram
Author: Markus Ressel
Author-email: mail@markusressel.de
License: AGPLv3+
Location: /home/user/.local/lib/python3.10/site-packages
Requires: click, container-app-conf, emoji, python-telegram-bot
Required-by:
[~]$

[~]$ pip show click container-app-conf emoji python-telegram-bot > cli.log

Name: click
Version: 8.1.3
Summary: Composable command line interface toolkit
Home-page: https://palletsprojects.com/p/click/
Author: Armin Ronacher
Author-email: armin.ronacher@active-4.com
License: BSD-3-Clause
Location: /home/user/.local/lib/python3.10/site-packages
Requires:
Required-by: cli2telegram
Name: container-app-conf
Version: 5.2.2
Summary: Convenient configuration of containerized applications
Home-page: https://github.com/markusressel/container-app-conf
Author: Markus Ressel
Author-email: mail@markusressel.de
License: MIT
Location: /home/user/.local/lib/python3.10/site-packages
Requires: py-range-parse, python-dateutil, pytimeparse, ruamel.yaml, ruamel.yaml.clib, six, toml, voluptuous
Required-by: cli2telegram
Name: emoji
Version: 2.2.0
Summary: Emoji for Python
Home-page: https://github.com/carpedm20/emoji/
Author: Taehoon Kim, Kevin Wurster and Tahir Jalilov
Author-email: carpedm20@gmail.com
License: New BSD
Location: /home/user/.local/lib/python3.10/site-packages
Requires:
Required-by: cli2telegram
Name: python-telegram-bot
Version: 13.15
Summary: We have made you a wrapper you can't refuse
Home-page: https://python-telegram-bot.org/
Author: Leandro Toledo`

**Comment by markusressel** [comment]
Meh, the same issue but for emojize. This issue has been addressed already in #13, however I think there hasn't been a release since then so the change is not part of the v2.0.0 release.
You can also downgrade emojize to [1.7.0](https://github.com/carpedm20/emoji/releases/tag/v1.7.0) to get this running for now. Of course this is only a temporary solution, and a new release would be necessary to correct this.

**Comment by maksimaee** [comment]
Thank you, this got it working!

**Comment by markusressel** [comment]
I have just released 2.0.1, which should fix all of the issues.
Feel free to give it a try! :rocket: :nerd_face:
